src/index.ts

```diff
@@ -1,9 +1,10 @@
+import Ajv from 'ajv';
 import type { FastifySchema, FastifySchemaCompiler, FastifyTypeProvider } from 'fastify';
 import type { FastifySerializerCompiler } from 'fastify/types/schema';
-import type { z, ZodAny, ZodTypeAny } from 'zod';
+import type { z, ZodAny, ZodError, ZodTypeAny } from 'zod';
 import { zodToJsonSchema } from 'zod-to-json-schema';
+import { fromZodError } from 'zod-validation-error';
```

Comment:
this change might be useful, but it seems to be unrelated to the issue with mercurius. Can you create a separate PR for that?

Reply:
After this PR, I will do it